.github/label-actions.yml

```
@@ -217,15 +217,57 @@
     Thanks, the Renovate team
   close: true
   close-reason: 'not planned'
-  lock: true
-  lock-reason: 'resolved'
 
-'format-code-comments':
+'auto:format-code':
   comment: >
     Hi, please format any copy/pasted code or logs so they're readable.
 
 
     You can find a Markdown code formatting guide [here](https://www.markdownguide.org/basic-syntax/#code) as well as some GitHub-specific information formatting code blocks [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).
 
 
+    Thanks, the Renovate team
+
+'auto:discussion-closed':
+  comment: >
+    Hi there,
+
+
+    This discusion is being closed because it's missing some details, or something else about it has made it difficult or impossible for us to help resolve the issue.
+
+
+    We choose to close and later delete such discussions in order to keep a high quality of discussions in this repository for users who search for answers.
+
+
+    You are welcome to create a new discussion with more details in future.
+
+
+    Thanks, the Renovate team.
+  close: true
+  close-reason: 'not planned'
+
+'auto:bad-vibes':
+  comment: >
+    Hi there,
+
+    A maintainer has flagged this discussion as giving off some bad vibes. **Maintainers will no longer participate in this discussion unless/until such "vibes" are resolved.**
+
+
+    This is typically when someone is overly critical of the product or the assistance they're receiving, or being unreasonably demanding or lazy, such as refusing to provide logs or other information which they're asked.
+
+
+    Open Source means you're welcome to use the software with few limits applied, but it does not mean you are welcome to give maintainers a bad day or contribute to Open Source burnout.
+
+
+    If you feel this is in error or a misunderstanding, please let us know. If you see this message but don't take it as a sign to cool off or back off, we may close the discussion and block you from the repository.
+
+
+    If you see this message and want to get the discussion back on track, it would be appreciated if you can:
+    - Edit any previous comments which may have given off any bad vibes
+    - Acknowledge that you understand the problem (no lengthy apology is needed, we just want to know it won't continue)
```

```suggestion
    If you want to get the discussion back on track, we would like you to:
    - Remove, or edit, the bad vibes parts of your comments
    - Give a short apology
    - Stop giving off more bad vibes
```

We can drop the "If you see this message" as it's redundant. Those reading the message obviously "see it". If the bad actor ignores the message and discussion, we're done also.

We should be very clear on the wanted behavior from the user after this message.

---

I actually _don't_ want an apology in most cases, especially if the bad vibes are unintended. I just want an acknowledgment

---

How about this then?

```suggestion
    If you want to get the discussion back on track, we would like you to:
    - Remove, or edit, the bad vibes parts of your comments
    - Say that you understand the problem (we don't need a lengthy apology)
    - Stop giving off more bad vibes
```